core/src/main/scala/cats/instances/order.scala

```diff
@@ -25,7 +25,6 @@ package instances
 import cats.kernel.instances.unit._
 
 trait OrderInstances extends kernel.instances.OrderInstances {
-
   implicit val catsContravariantMonoidalForOrder: ContravariantMonoidal[Order] =
```

Comment:
can we not do `Decidable[Order]`, `Decideable[Hash]` and `Decidable[Ordering]`? Seems doable (well, maybe Hash isn't totally clear, not sure).

Would be interesting to grep for `ContravariantMonoidal` and see how many instances are actually Decidable.

Reply:
We actually had `Order` and `Ordering` at one point, I believe I removed them based on feedback from a prior iteration, but those concerns may be gone now. I am happy to re-add them. (In practice almost all `ContravariantMonoidal` that are not derived from a base `ContravariantMonoidal` are `Decidable` it appears to me, but I'm sure in the wild there are more cases where these come apart than in the base).

Reply:
Ah I remember now what the deal with `Order` and `Ordering` does - the natural lexicographic approach to ordering the eithers causes the instance to fail the `right sum distributivity` law we've defined in this changeset. I'll probably leave them out since I don't have a better idea on how that would be defined, but maybe `Hash` is more fortunate.